hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/utils/db/DBConfigFromFile.java

```diff
@@ -65,7 +65,7 @@ public static File getConfigLocation() throws IOException {
 
     if (StringUtil.isBlank(path)) {
```

Comment:
We duplicate this if statement here, first to write a debug level log message second to return null, can we pull the two together, and also change the log message here, and state the reason why we fail by saying something like:
CONFIG_DIR + " variable is empty, please make sure it is setup correctly!"

Reply:
Not introduced by this change, but thanks for pointing out, let's fix it anyway.